core/src/main/kotlin/org/evomaster/core/search/service/Archive.kt

```diff
@@ -458,4 +459,14 @@ class Archive<T> where T : Individual {
     fun getSnapshotOfSamplingCounter() : Map<Int, Int>{
         return samplingCounter
     }
+
+    fun exportCoveredTargetsAsPair(solution: Solution<*>) : List<Pair<String, List<Int>>>{
```

Comment:
provide comments specifying what is returned.
Eg, in `Pair`, what is the `String` and `List<Int>` representing?

Reply:
fixed

Reply:
how was this "fixed"? I do not see any JavaDocs explaining the return type